ci/tasks/lnd-smoketest.sh

```diff
@@ -4,20 +4,17 @@ set -eu
 
 source smoketest-settings/helpers.sh
 
-lndmon_host=`setting "lndmon_endpoint"`
 lnd_host=`setting "lnd_p2p_endpoint"`
 
-if [[ "${lndmon_host}" != "" ]]; then
+if [[ "${lnd_host}" != "" ]]; then
   set +e
   for i in {1..60}; do
-    echo "Attempt ${i} to curl lndmon"
-    curl -f ${lndmon_host}:9092/metrics
+    echo "Attempt ${i} to connect the lnd_p2p_endpoint"
+    nc -zv ${lnd_host} 9735
```

Comment:
Is checking the p2p port connectivity equivalent to the previous smoketest?

Which port does lndmon use to connect to lnd to gather metrics? iirc it was the rpc port. If so, then we should also test connectivity to the rpc port as our other services directly rely on that.

Reply:
In the lndmon chart the GRPC port was hardcoded as: `--lnd.host={{.Release.Name}}:10009` so changed to test that first and proceed to the p2p endpoint.